content/c-sharp/concepts/strings/terms/toupper/toupper.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: '.ToLower()'
+Description: 'The ToUpper() method is used to convert all the characters in a given string to uppercase.'
+Subjects:
+- 'Computer Science'
+- 'Code Foundations'
+  Tags:
+- 'Strings'
+- 'Methods'
+  CatalogContent:
+- 'learn-c-sharp'
+- 'paths/computer-science'
+---
+
+The **`ToUpper()`** method is a method in C# that takes a string and returns a copy of this string converted to uppercase. Numbers and symbols are not changed.
+
+## Syntax
+
+```pseudo
+string.ToUpper()
+```
+The .ToUpper() return a new string whose letters are converted to uppercase.
+
+This method will not change the input string.
+
+
+## Example
+
+In this following example, `originalString` is the string being converted to uppercase, and `uppercaseString` is the new string that contains the uppercase characters. We can see that that the original string remains unchanged:
+
+```cs
```

Comment:
please include the code setup, the class and main calls, in the first example as well. It should be runnable as is.